.github/CONTRIBUTING.md

```diff
@@ -1,18 +1,24 @@
 # Contributing
 
-Before contributing a new component to Dialtone Vue, please check the [Jira board](https://switchcomm.atlassian.net/jira/software/projects/DT/boards/187/backlog) to see if there is already an issue for it and if not, add it.
+Before contributing a new component to Dialtone Vue, please check the [Jira board](https://switchcomm.atlassian.net/jira/software/projects/DT/boards/187/backlog)
+to see if there is already an issue for it and if not, add it.
 
 ## Overview
 
 ### What Is Dialtone Vue?
 
-Dialtone is a design system by Dialpad comprised of CSS components, Vue components, utility classes, visual assets, documentation. Dialtone Vue is the Vue component portion of Dialtone and is distributed as a separate package. It is recommended that you are familiar with [Dialtone](https://github.com/dialpad/dialtone/tree/staging) as well as it's [contribution guide](https://github.com/dialpad/dialtone/blob/staging/.github/CONTRIBUTING.md) before you contribute to Dialtone Vue.
+Dialtone is a design system by Dialpad comprised of CSS components, Vue components, utility classes, visual assets, documentation.
+Dialtone Vue is the Vue component portion of Dialtone and is distributed as a separate package.
+It is recommended that you are familiar with [Dialtone](https://github.com/dialpad/dialtone/tree/staging)
+as well as it's [contribution guide](https://github.com/dialpad/dialtone/blob/staging/.github/CONTRIBUTING.md) before you contribute to Dialtone Vue.
 
-Dialtone is a dependency of Dialtone Vue.
+Dialtone is a peerDependency of Dialtone Vue.
```

Comment:
since we aren't referencing actual code we can write the words "peer dependency"